src/components/designSystem/Status.tsx

```diff
@@ -178,7 +180,7 @@ export const Status: FC<StatusProps> = ({
       data-test="status"
     >
       <Typography variant="captionHl" color={config.color} noWrap>
-        {translate(statusLabel, labelVariables ?? {})}
+        {typeof label === 'number' ? label : translate(statusLabel, labelVariables ?? {})}
```

Comment:
_Nitpicking_
I wonder if doing this could be a better alternative: 
`{typeof label === 'string' ? translate(statusLabel, labelVariables ?? {}) : label}`
So in that case, if later we want to update the type to accept a `ReactNode`, we won't have to update the template too.